dashboard/src/actions/catalog.ts

```diff
@@ -60,16 +62,44 @@ const actions = [
 
 export type ServiceCatalogAction = ActionType<typeof actions[number]>;
 
+function isEmptyDeep(obj: any): boolean {
+  if (typeof obj === "number") {
```

Comment:
why is number a special case? What about string and others?

Reply:
it's a `lodash` problem:

https://github.com/lodash/lodash/blob/4ea8c2ec249be046a0f4ae32539d652194caf74f/isEmpty.js#L33

In our case, we want to mark a number as non empty so we need this special case. It's not critical though because you cannot set a "number" in a form (AFAIK) but just in case.